frontend/src/styles/utilities.scss

```diff
@@ -957,3 +957,21 @@
 .sticky {
     position: sticky;
 }
+.line-clamp-2 {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+}
+.object-cover {
+    object-fit: cover;
+}
+.object-center {
+    object-position: center;
+}
+.h-28 {
```

Comment:
You probably already know most of this: We're using this utility file to add a subset of tailwindcss classes. This is useful, because many people have prior knowledge of tailwind and we can use IDE plugins for autosuggest / intellisense. Therefore we're trying to stay close to the actual tailwind implementation.

To add a new size for heights, I'd suggest you actually add it to the `$tiny_spaces` or `$humongous_spaces` in `src/styles/vars.css`. See here for a reference of supported tailind styles: https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale. 

`h-28` would actually be `112px`.  You can derive this by dividing by 4, which gives you the rem. In this case 28 / 4 = 7rem. 1rem is 16px for us.

minor nit: a new line between unrelated utilities would be nice